snippets/csharp/System.Linq/Enumerable/AggregateTSource/enumerable.cs

```diff
@@ -4,19 +4,16 @@
 
 namespace SequenceExamples
 {
-    class Program
+    static class Program
     {
         // This part is just for testing the examples
-        static void Main(string[] args)
-        {
-            OfTypeEx1();
-        }
+        static void Main() => OfTypeEx1();

```

Comment:
If we're embracing modern C# shouldn't this be changed to use top-level statements instead?

Reply:
Yes. We'd prefer top-level statements.

We don't enforce the change on PRs, because we have many samples that pre-date top-level statements. However, if the purpose of this PR is to modernize the code, it should be top-level statements.